### ISSUE TITLE
[Accessibility]: Notebooks read-only duplicate labels in search block 

### ISSUE THREAD

**Issue by marisak** [sourcegraph/sourcegraph#35674]
### Audit type

ARC Toolkit

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/35142

### Problem description

The test for `duplicateLabelsUsed` failed. I'm not exactly sure what's going on, but it appears that there's a conflict between the `<body>` tags used for the entire page, and also potentially `<body>` tags used in the search results. 

### Expected behavior

Each text label should be unique. 

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart. I have assigned the label 'gitstart' and added the assignee 'gitstart-sourcegraph'.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.

**Comment by gitstart-sourcegraph** [comment]
Hello @marisak 
Trying to reproduce the failing test as outlined in this issue. Using the ARC Tookit to run test on Notebooks list and Edit page, however `duplicateLabelsUsed` is not outlined as one of the failing test. This might have been resolved as a result of other fixes, or perhaps we need to test differently to reproduce this issue. Kindly confirm?
Here are some attached images for carried out test
CC: @limitedmage

<img width="1440" alt="Screen Shot 2022-06-09 at 20 05 30" src="https://user-images.githubusercontent.com/89894075/173028037-60fb479a-7dfd-465a-b569-cd2a93f345b6.png">
<img width="1440" alt="Screen Shot 2022-06-09 at 20 06 38" src="https://user-images.githubusercontent.com/89894075/173028080-92102674-fa58-4888-a006-2083c8579d9b.png">

**Comment by limitedmage** [comment]
It might be from this? https://github.com/sourcegraph/sourcegraph/pull/36643 cc @umpox

**Comment by limitedmage** [comment]
Either way, if this is no longer present, this can be closed. Please reopen if you discover otherwise.
